swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -7834,11 +8007,11 @@ components:
       type: object
       additionalProperties: 
         $ref: "#/components/schemas/hrefType"
-      required:
-        - cardAccount
       properties:
         cardAccount:
           $ref: "#/components/schemas/hrefType"
+        card:
+          $ref: "#/components/schemas/hrefType"
```

Comment:
Relevant Change: card is new. We already have it
Adoption approach: Nothing to change